draft-irtf-cfrg-hpke.md

```diff
@@ -219,8 +219,8 @@ informative:
 
   TestVectors:
     title: "HPKE Test Vectors"
-    target: https://github.com/cfrg/draft-irtf-cfrg-hpke/blob/3d6ced124134825ed7a953b126cf5f756d960bc9/test-vectors.json
-    date: 2020
+    target: https://github.com/cfrg/draft-irtf-cfrg-hpke/blob/779d0285fe0e9407abb549ba0104e831d9677164/test-vectors.json
```

Comment:
Note that this file isn't a JSON file, despite the name.  Maybe https://github.com/cfrg/draft-irtf-cfrg-hpke/raw/779d0285fe0e9407abb549ba0104e831d9677164/test-vectors.json instead?